sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Property `CompletedOn` renamed to `TrainingCompletedOn` on `CustomFormModel` and `CustomFormModelInfo`.
 - Property `LabelText` renamed to `LabelData` on `FormField`.
 - Property `ValueText` renamed to `ValueData` on `FormField`.
+- Property `Type` renamed to `ValueType` on `FieldValue`.
```

Comment:
This is in the wrong release.

Reply:
I got to excited with all the renames. 
Thanks for catching this!